### ISSUE TITLE
how to write this ntfs img to usb disk

### ISSUE THREAD

**Issue by eocene317** [mtivadar/windows10_ntfs_crash_dos#1]

**Comment by omod** [comment]
@eocene317 

I tried to use `dd`,  but it didn't work on `Windows7 Pro 6.1.7601 SP1, Build 7601 x64`.

```
➜ sudo dd if=./tinyntfs of=/dev/disk[N] bs=1m
```

> N is the udisk physical device number

**Comment by cxaqhq** [comment]
写入u盘成功了，但是我插在windows10上面没有反应啊，windows10提示”无法访问，文件或目录损坏且无法访问“是什么情况？

**Comment by cxaqhq** [comment]
sudo dd if=./tinyntfs of=/dev/sdb1
[sudo] password for cx: 
20472+0 records in
20472+0 records out
10481664 bytes (10 MB, 10 MiB) copied, 3.76947 s, 2.8 MB/s

**Comment by richard1225** [comment]
anybody succeed？

**Comment by cxaqhq** [comment]
没有啊，我就没有成功
@richard1225   I did not succeed

**Comment by Twml1500** [comment]
@mitsukuri I get "File write failed: access denied" error with restore from file option.

**Comment by mitsukuri** [comment]
@Twml1500 Running it as admin? Any filesystem activity on the device?

**Comment by Twml1500** [comment]
@mitsukuri what filesystem should my usb stick be? maybe that matters

**Comment by lvguanjie** [comment]
can't work .how to use

**Comment by dogtopus** [comment]
I cannot reproduce this on latest Windows 8.1 Industry Pro either. By the age of this expoit (reported to Microsoft on July 2017) it is safe to assume that Microsoft already fixed it silently.

**Comment by Twml1500** [comment]
@mtivadar 

dd.exe if=tinyntfs of=\.\d: bs=1M count=10

rawwrite dd for windows version 1.0beta1 WIN64.
Written by John Newbigin <jnewbigin@chrysocome.net>
This program is covered by terms of the GPL Version 2.

Error opening output file: 123 The filename, directory name, or volume label syntax is incorrect.

tinyntfs is in the directory the command was ran from

**Comment by mtivadar** [comment]
please see readme

**Comment by Twml1500** [comment]
finally got the files to write to a usb with.
dd if=tinyntfs of=\\.\d:

plugging in the usb after writing the drive shows up as D: in explorer with size of 9.99mb
clicking on drive says: "The file or directory is corrupt and unreadable" 

nothing autoruns, no bsod.

**Comment by mtivadar** [comment]
Make sure you pulled the last version of tinyntfs

**Comment by Twml1500** [comment]
working for me now wrote on windows with dd.exe

download from here(used beta5)
http://www.chrysocome.net/downloads/1f150e835d6bde24e2fc1c9107e8e503/dd-0.4beta5.zip

run command as admin in cmd: 
dd if=tinyntfs of=\\.\d:

(dual \ after of= before the .  for some reason i cant get a dual \ to display here)
(note d: is the drive letter of my NTFS formatted usb stick)

as soon as you plug in the stick explorer tries to open, then instant bsod ntfs.sys is the cause.

**Comment by shotbyapony** [comment]
Anyone help me? this is what I'm getting:
`rawwrite dd for windows version 0.4beta5.
Written by John Newbigin <jn@it.swin.edu.au>
This program is covered by the GPL.  See copying.txt for details
Error opening output file: 32 The process cannot access the file because it is being used by another process`

**Comment by mtivadar** [comment]
Creating an autotun.inf with icon=4 should crash immediately

**Comment by Anton238** [comment]
@Twml1500 If you have not figured out the access error yet, then you should go into the properties of the flash drive and give full access to your user (or for all) on the security and access tab. [Link to the instructions in the pictures](https://photos.app.goo.gl/QXjOMvq0gKaf2noq2)

**Comment by UnforeseenOcean** [comment]
You can add `autorun.inf` by writing this to that file (you only need `icon=4` part for this to work):
```
[autorun]
icon=4
open=FAKEFILE.EXE
label=BSODSTICK
```
Then dragging the file into the drive icon on Windows 7 or by using Linux.
To test, just unplug the device and replug it if you are on Windows 7.

**Comment by UnforeseenOcean** [comment]
@shotbyapony What is your OS? Do you have `autorun.inf`? 
With it, it crashes almost immediately every time, and on the same computer.

**Comment by abhimanyuZ** [comment]
@UnforeseenOcean will you able to recover your windows after getting the BSOD?

**Comment by XxTWMLxX** [comment]
@abhimanyuZ the ntfs BSOD is a harmless hard stop error, 
reboot and pc will start normally. no long term damage.

**Comment by UnforeseenOcean** [comment]
@abhimanyuZ Yes, it's a harmless BSOD and unless there was some file that was being written it should be okay.
I sometimes use this to prevent people from accessing my computer by plugging this in when I leave my seat for a long time.

**Comment by abhimanyuZ** [comment]
@UnforeseenOcean  @XxTWMLxX 
Then in which way this exploit can actually cause trouble to victim?

**Comment by UnforeseenOcean** [comment]
@abhimanyuZ The crash occurs immediately when the OS recognizes and tries to access the disk in question; it can be used as a way to cause denial-of-service.
Now I wonder though, will this work with virtual disk colutions?

**Comment by XxTWMLxX** [comment]
@abhimanyuZ it can be used to generate a crash dump file that can contain the bitlocker encryption key in it, with enough knowledge and time you can extract that info and get the bitlocker password, then be able to steal and use the pc/data on it.   just one of the many uses, it can also be combined with malware like a rat, rat installs,bsod happens, malicious person(s) then can remotely recover the before mentioned crash dump.
